### ISSUE TITLE
训练时lr的显示问题

### ISSUE THREAD

**Issue by CJ666-njust** [WZMIAOMIAO/deep-learning-for-image-processing#726]
up你好，在模型的训练过程中，lr的显示仅在warm up的过程会变化，之后就固定了。但是tensorboard中记录的lr却是正常的。是create_lr_schedule函数中的返回值的问题吗

**Comment by HaronCHou** [comment]
lr不是固定的吧？是有函数的呢，有学习率update。up用的Cos函数的lr下降呀。`scheduler.step()`

**Comment by WZMIAOMIAO** [comment]
能具体说下是那个模型的训练脚本，然后你学习率相关的设置是怎样的，epoch设置的是多少。
